### PR TITLE
Adding projects from the manually added projects on Code.gov

### PR DESCRIPTION
**Pull request by jordangov**
Making the code.json inventory file include what Code.gov had manually thrown together before we had a proper JSON file.

### CODE REVIEW

**Comment by jordangov** [comment]
Can I get a quick check @jgarber623-gov ? This should just be a JSON inventory file update, but a quick look would be great.

File: code.json [hunk 1]
```diff
@@ -7,9 +7,12 @@
 	"releases": [
 		{
 			"organization": "NCI Agency",
-			"name": "Advisor Network",
+			"name": "ANET",
 			"description": "The Advisor Network (ANET) is a tool to track relationships between advisors and advisees. ANET was initially built by the Defense Digital Service in support of the USFOR-A and Resolute Support mission to train, advise, and assist the Afghan government.",
-			"tags": [],
+			"tags": [
+				"advisor",
+        "network"
```
Comment (jgarber623-gov) [inline code.json]: Fix indentation.

File: code.json [hunk 2]
```diff
@@ -217,8 +224,87 @@
...
+        "interface",
+        "military",
+        "courts",
+        "martial",
+        "manual"
+      ],
+      "languages": [
+        "HTML",
+				"JavaScript",
+				"CSS"
```
Comment (jgarber623-gov) [inline code.json]: Fix indentation.

File: code.json [hunk 3]
```diff
@@ -217,8 +224,87 @@
...
+            "URL": "https://github.com/deptofdefense/eMCM/blob/master/LICENSE.md",
+            "name": "AGPL"
+          }
+        ],
+        "usageType": "openSource"
+      },
+      "laborHours": 1,
+      "date": {
+        "created": "2016-12-14",
+				"lastModified": "2017-10-10T16:12:00Z",
```
Comment (jgarber623-gov) [inline code.json]: Fix indentation.

File: code.json [hunk 4]
```diff
@@ -217,8 +224,87 @@
...
+      }
+    },
+    {
+      "status": "Production",
+      "vcs": "git",
+      "name": "Dshell",
+      "description": "An extensible network forensic analysis framework. Enables rapid development of plugins to support the dissection of network packet captures.",
+      "contact": {
+        "email": " ",
+        "name": ""
```
Comment (jgarber623-gov) [inline code.json]: Do we have this information? If not, recommend remove the `"contact"` section.

Comment (jordangov) [reply code.json]: Nope, this project was taken from Code.gov's existing (manual) JSON file creation.

**Comment by jordangov** [comment]
Gah... stupid indentation. Atom was supposed to auto-detect and do this. Maybe we need to add a JSON beautifier.

**Comment by jordangov** [comment]
Okay... another pass @jgarber623-gov or @arichiv-usds ?

**Comment by jgarber623-gov** [comment]
@jordangov I use the Atom [linter-jsonlint package](https://atom.io/packages/linter-jsonlint) which is super helpful for catching JSON formatting issues.

**Pull request merged**